public/sanda/practice/sanda4.5.html

```diff
@@ -53,7 +53,7 @@
                 'Buiucani',
                 'Botanica'
             ];
-            return districts[generateRandomIntInRange(0, 5)];
+            return districts[generateRandomIntInRange(0, length)];
```

Comment:
This doesn't work correctly.
`length` is a property of array, but you use it as a variable here.

Please, check the example and fix:
https://www.w3schools.com/jsref/jsref_length_array.asp